velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -290,7 +290,7 @@ TEST_F(HdfsFileSystemTest, schemeMatching) {
   try {
     auto fs = std::dynamic_pointer_cast<filesystems::HdfsFileSystem>(
         filesystems::getFileSystem("/", nullptr));
-    FAIL() << "expected VeloxException";
```

Comment:
@wypb why this change?

Reply:
Looks like an exception was previously thrown but not anymore.

Reply:
Because `InsertIntoHdfsTest` relies on `HiveConnectorTestBase`, and the `HiveConnectorTestBase` constructor will register LocalFileSystem (`filesystems::registerLocalFileSystem()`), so when executing the `HdfsFileSystemTest.schemeMatching` test, `filesystems::getFileSystem("/", nullptr)` will not get an error, but the `LocalFileSystem` is obtained, so `auto fs =std::dynamic_pointer_cast( filesystems::getFileSystem("/", nullptr));` will not get an error, but get a null pointer.

Reply:
@wypb It is not ideal that a test behaves differently in different settings.
I will fix the LocalFileSystem registration.

Reply:
No problem, I'll fix it.

Reply:
Hi @majetideepak  I modified the order of the `InsertIntoHdfsTest` and `HdfsFileSystemTest` test sets in `velox_hdfs_file_test`. Now the test set in `HdfsFileSystemTest` will be run first, and then `InsertIntoHdfsTest`. so the registration of `LocalFileSystem` will not be executed until `HdfsFileSystemTest` runs, this will ensure that the test set in `HdfsFileSystemTest` runs correctly.

Reply:
Change from
```
add_executable(
  velox_hdfs_file_test InsertIntoHdfsTest.cpp HdfsFileSystemTest.cpp
                       HdfsMiniCluster.cpp HdfsUtilTest.cpp)
```
to
```
add_executable(
  velox_hdfs_file_test HdfsFileSystemTest.cpp InsertIntoHdfsTest.cpp
                       HdfsMiniCluster.cpp HdfsUtilTest.cpp)
```